cmd/gentypes/gentypes.go

```diff
@@ -420,6 +429,24 @@ package dap
 type Message interface {
 	GetSeq() int
 }
+// RequestMessage is an interface implemented by all Request-types. 
+type RequestMessage interface {
+	Message
+	// GetRequest provides access to the embedded Request-field by returning a pointer to it
```

Comment:
Here and below, simplify the method comment:

"GetRequest provides access to the embedded Request".

etc.